cmd/tableprinters/volume.go

```diff
@@ -13,9 +13,9 @@ func (t *TablePrinter) VolumeTable(data []*apiv1.Volume, wide bool) ([]string, [
 	var (
 		rows [][]string
 	)
-	header := []string{"ID", "Name", "Size", "Usage", "Replicas", "State", "StorageClass", "Project", "Partition"}
+	header := []string{"ID", "Name", "Size", "Usage", "Replicas", "State", "ClusterName", "StorageClass", "Project", "Partition"}
 	if wide {
-		header = []string{"ID", "Name", "Size", "Usage", "Replicas", "State", "StorageClass", "Project", "Partition", "Nodes"}
+		header = []string{"ID", "Name", "Size", "Usage", "Replicas", "State", "ClusterName", "StorageClass", "Project", "Partition", "Nodes"}
```

Comment:
I would remove the State column

Reply:
![image](https://github.com/user-attachments/assets/b0fd0e3a-a342-443a-b244-919010e86110)